relay-filter/src/browser_extensions.rs

```diff
@@ -107,8 +107,15 @@ fn get_exception_value(event: &Event) -> Option<&str> {
 fn get_exception_source(event: &Event) -> Option<&str> {
     let exception = get_first_exception(event)?;
     let frames = exception.stacktrace.value()?.frames.value()?;
-    let last_frame = frames.last()?.value()?;
-    Some(last_frame.abs_path.value()?.as_str())
+    // Iterate from the tail and get the first frame which is not <anonymous>.
+    for f in frames.iter().rev() {
+        let abs_path = f.value()?.abs_path.value()?;
+        let path = abs_path.as_str();
+        if path != "<anonymous>" {
```

Comment:
_nit_: You could introduce a constant for this and in the comment on the constant add a sentence or two on why we're skipping anonymous.